src/components/Best-selling-product/best-selling.jsx

```diff
@@ -0,0 +1,128 @@
+import { React, Fragment } from "react";
+import "./best-selling.css";
+export function BestSelling() {
+  return (
+    <Fragment>
+      <div class="best-seller-head head-font mgn-lt-4">
+        Best selling products
+      </div>
+
+      <div class="flex flex-wrap best-selling col-gap-2 row-gap-2 jc-center align-i-center">
+        <div class="product-cards flex flex-direct-col jc-sb">
+          <div class="product-display-img-container mgn-tp-0-5 mgn-rt-auto mgn-lt-auto">
```

Comment:
IMP Change ```class``` to ```className ```

Reply:
Sure, forgot to change that, will do it asap